File: Selective fear of numbers/Selective fear of numbers.py
def am_I_afraid(day,num):
    if day=="Monday":
        return num==12
    elif day=="Tuesday":
        return num >95
    elif day=="Wednesday":
        return num==34
    elif day=="Thursday":
        return num==0
    elif day=="Friday":
        return num%2==0
    elif day=="Saturday":
        return num==56
    else:
        return num==666 or num==-666

File: Selective fear of numbers.js
const AmIAfraid = function(day, num){
  return {
        'Monday':  num == 12,
        'Tuesday': num > 95,
        'Wednesday': num == 34,
        'Thursday': num == 0,
        'Friday': num % 2 == 0,
        'Saturday': num ==  56,
        'Sunday': num == 666 || num == -666,
    }[day]
}
